src/OmniSharp.Roslyn.CSharp/Services/Completion/CompletionListBuilder_Sync.cs

```diff
@@ -55,7 +55,7 @@ internal static partial class CompletionListBuilder
                     completionTasksAndProviderNamesBuilder.Add(((Task<CompletionChange>?)completionService.GetChangeAsync(document, completion), providerName));
                 }
             }
-            var completionTasksAndProviderNames = completionTasksAndProviderNamesBuilder.MoveToImmutable();
+            var completionTasksAndProviderNames = completionTasksAndProviderNamesBuilder.ToImmutable();
```

Comment:
@filipw why did you revert this? Just to see if it would fix the OOM issue?

Reply:
it failed many tests with `System.InvalidOperationException : MoveToImmutable can only be performed when Count equals Capacity.`